src/styles/blocks/card.scss

```
@@ -0,0 +1,115 @@
+.card {
+  width: $widthCard; // 200px
```

no need for the comments here.
Imagine the page was redesigned - what will you do? You will need to fix all the comments manually. This way variable makes no sense if you need to fix it manually in the code 😄 
```suggestion
  width: $widthCard;
```